src/front/app/components/Categories/index.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { getCategories, toggleCategory } from './../../actions/categories';
+import Category from './Category';
+
+class CategoryList extends Component {
+    constructor(props) {
+        super(props);
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+    componentDidMount() {
+        const { dispatch } = this.props;
+        dispatch(getCategories());
+    }
+
+    handleClick(name) {
+        const { dispatch } = this.props;
+
+        dispatch(toggleCategory(name));
+    }
+
+    render() {
+        const { categories } = this.props;
+
+        return (
+            <ul>
+                {categories.map((category, index) => (
+                    <Category
+                      key={index}
+                      {...category}
+                      onClickHandler={() => {this.handleClick(category.name);}}
```

Comment:
I've already remove it, but i will commit it with implemented toggle category functionality. It has not implemented yet. This pull request related only to displaying categories